viper.go

```diff
@@ -1280,6 +1287,14 @@ func (v *Viper) find(lcaseKey string, flagDefault bool) interface{} {
 		return nil
 	}
 
+	if v.overriders != nil && len(v.overriders) != 0 {
```

Comment:
Shouldn't this actually be a reverse search? Start at the end and work backwards until you find a value in an overrider? So it would be last overrider in wins. Otherwise you can't override and overrider.

Reply:
Yeah good idea